xds/internal/client/client.go

```diff
@@ -229,6 +239,8 @@ type ClusterUpdate struct {
 	EnableLRS bool
 	// SecurityCfg contains security configuration sent by the xDS server.
 	SecurityCfg *SecurityConfig
+	// Thresholds contains the circuit breaker thresholds, if any.
+	Thresholds []CircuitBreakerThreshold
```

Comment:
Make this a `*CircuitBreakerThreshold`, and keep only the one with priority `DEFAULT`.

Reply:
Since the logic for iterating over the thresholds and using the first value with priority `"DEFAULT"` is moved here, `Thresholds` was replaced with `MaxRequests` which is a pointer to the max requests if found.